### DISCUSSION TITLE
Allow deploy Ruby applications using Devopness

### DISCUSSION THREAD

**Discussion by jfoliveira** [devopness/devopness#837] [Ideas / Feature requests]
### Problem to be solved
Devopness already support multiple application stacks allowing deploy application written in multiple programming languages.
However, at the time of this writing Ruby is not supported, so developers that deploy applications using Ruby cannot take advantage of using Devopness as a single platform for multi-cloud infrastructure and application deployment for all their existing applications.

### Why does this matter?
According to [1] and [2] Ruby is a very popular programming language, specially among web developers, hence having Ruby as an application stack supported by Devopness would help developers to use a single solution for deploying their Ruby applications in a productive and reliable way.

[1] [TIOBE index](https://www.tiobe.com/tiobe-index/) says Ruby is among the top 20 most popular programming languages, as of December 2023

<img width="812" alt="image" src="https://github.com/devopness/devopness/assets/1943318/f41d97e4-19da-4788-90f5-8bdecd3d7a20">

[2] [StackOverflow 2023 Developer Survey](https://survey.stackoverflow.co/2023/#technology-most-popular-technologies) says Ruby is amongst the top 10 most popular programming languages adopted by web developers

<img width="810" alt="image" src="https://github.com/devopness/devopness/assets/1943318/2b1b22b9-9f52-4af5-95e7-ec0db019d99b">

**Comment by jfoliveira** [comment]
Added support to `Ruby` as a supported application stack.
Devopness users can now deploy applications using Ruby and choose between versions `3.2.2` and `3.1.4`:
<img width="214" alt="image" src="https://github.com/devopness/devopness/assets/1943318/f5be4e2e-44c9-4794-8fc5-f80f7074ccc3">

Leaving this discussions open while we wait for beta testers to confirm they could successfully deploy Ruby applications with no issues.

**Reply by devopness-social** [reply]
Many users are already deploying Ruby applications with Devopness and also Ruby on Rails apps.
See video below for a quick reference.


### Deploy Ruby on Rails to any cloud provider (AWS, Azure, GCP and more) using Devopness

[![Deploy Ruby on Rails to any cloud provider (AWS, Azure, GCP and more) using Devopness ](https://i.ytimg.com/vi/pVjtBu5_lJI/maxresdefault.jpg)](https://www.youtube.com/watch?v=pVjtBu5_lJI "Deploy Ruby on Rails to any cloud provider - Click to watch!")
